sky/packages/sky/lib/src/widgets/framework.dart

```diff
@@ -1170,6 +1194,7 @@ abstract class RenderObjectElement<T extends RenderObjectWidget> extends Element
       _ancestorRenderObjectElement.removeChildRenderObject(renderObject);
       _ancestorRenderObjectElement = null;
     }
+    _slot = null;
```

Comment:
be consistent, in this and the superclass implementation of this method, about whether _slot is nulled out first or last.